pkg/server/backend_manager.go

```diff
@@ -35,40 +35,58 @@ import (
 	"sigs.k8s.io/apiserver-network-proxy/proto/header"
 )
 
-type ProxyStrategy string
+type ProxyStrategy int
 
 const (
 	// With this strategy the Proxy Server will randomly pick a backend from
 	// the current healthy backends to establish the tunnel over which to
 	// forward requests.
-	ProxyStrategyDefault ProxyStrategy = "default"
+	ProxyStrategyDefault ProxyStrategy = iota + 1
 	// With this strategy the Proxy Server will pick a backend that has the same
 	// associated host as the request.Host to establish the tunnel.
-	ProxyStrategyDestHost ProxyStrategy = "destHost"
-
+	ProxyStrategyDestHost
 	// ProxyStrategyDefaultRoute will only forward traffic to agents that have explicity advertised
 	// they serve the default route through an agent identifier. Typically used in combination with destHost
-	ProxyStrategyDefaultRoute ProxyStrategy = "defaultRoute"
+	ProxyStrategyDefaultRoute
 )
 
+func (ps ProxyStrategy) String() string {
+	return [...]string{"default", "destHost", "defaultRoute"}[ps-1]
```

Comment:
This is not obviously correct.
Maybe something more like 
`
switch ps {
case ProxyStrategyDestHost: return "destHost"
case ProxyStrategyDefaultRoute: return "defaultRoute"
case ProxyStrategyDefault: return "default"
}
`

Reply:
Done, agreed it's more readable.